subprojects/docs/src/docs/userguide/jvm/jvm_test_suite_plugin.adoc

```diff
@@ -141,6 +141,14 @@ include::sample[dir="snippets/testing/test-suite-configure-suite-dependencies/ko
 <1> Configure the built-in `test` test suite.
 <2> Add the assertj library to the test's compile and runtime classpaths.  The `dependencies` block within a test suite is already scoped for that test suite.  Instead of having to know the global name of the configuration, test suites have a consistent name that you use in this block for declaring `implementation`, `compileOnly` and `runtimeOnly` dependencies.
 
+The `dependencies` block within a test suite does not (yet) provide access to the full set of dependency configuration methods that are available within a top-level `dependencies` block.
+For instance the link:{groovyDslPath}/org.gradle.api.artifacts.dsl.DependencyHandler.html#org.gradle.api.artifacts.dsl.DependencyHandler:platform(java.lang.Object)[platform()] and link:{groovyDslPath}/org.gradle.api.artifacts.dsl.DependencyHandler.html#org.gradle.api.artifacts.dsl.DependencyHandler:platform(java.lang.Object)[enforcedPlatform()] methods described in the section <<java_platform_plugin#sec:java_platform_bom_import,Sourcing constraints from another platform>> are not yet available.  You will need to use the equivalent way of configuring dependencies noted above to utilize these methods.
```

Comment:
But it will be pick up if you have defined the `platform` as a general dependency and only the actual artifacts on the suite?

Reply:
If I understand what you're asking properly, yes.  I just added a test to demonstrate this.

Reply:
Close, but what I meant was if I declared the platform in the regular dependencies block, will this affect the suites as well?
```
dependencies {
  api(platform("..."))
}
```